lib/parser/objects.ts

```diff
@@ -591,40 +598,48 @@ interface IVariableBase extends IReferenceable, IHasLexerToken, IHasType, IHasDe
   lexerToken: OLexerToken;
 }
 
-export class OFileVariable extends ObjectBase implements IVariableBase {
+export class OFileVariable extends ObjectBase implements IVariableBase, IReferenceable {
```

Comment:
`IVariableBase` already implements `IReferencable`
```suggestion
export class OFileVariable extends ObjectBase implements IVariableBase {
```